lib/app/modules/bulk_data/bulk_data_patient_export_sequence.rb

```diff
@@ -14,8 +14,26 @@ class BulkDataPatientExportSequence < SequenceBase
       requires :token
       conformance_supports :Patient
 
-      def check_export_kick_off(klass)
-        reply = export_kick_off(klass)
+      attr_writer :run_all_kick_off_tests
```

Comment:
It's safer to use an `attr_accessor` and only reference the accessors rather than the instance variable itself.